posthog/api/dashboards/dashboard.py

```diff
@@ -434,6 +446,16 @@ def move_tile(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         )
         return Response(serializer.data)
 
+    @action(methods=["POST"], detail=False)
+    def create_from_template_json(self, request: Request, *args: Any, **kwargs: Any) -> Response:
```

Comment:
🤔 this probably isn't blocking but leaves us with several ways of doing the same thing across duplicating insights, duplicating dashboards, creating them both, and creating them from templates.